include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -555,21 +627,60 @@ class alignment_algorithm :
             res.front_coordinate.first = trace_res.first_sequence_slice_positions.first;
             res.front_coordinate.second = trace_res.second_sequence_slice_positions.first;
 
-            if constexpr (result_config_t::rank == 3) // compute alignment
+            if constexpr (traits_t::result_type_rank == 3) // compute alignment
                 res.alignment = std::move(trace_res.alignment);
         }
 
         // Store the matrices in debug mode.
-        if constexpr (is_debug_mode)
+        if constexpr (traits_t::is_debug)
         {
             res.score_debug_matrix = std::move(score_debug_matrix);
-            if constexpr (result_config_t::rank == 3) // compute alignment
+            if constexpr (traits_t::result_type_rank == 3) // compute alignment
                 res.trace_debug_matrix = std::move(trace_debug_matrix);
         }
 
         return res;
     }
 
+    //!\overload
+    template <typename index_range_t, typename sequence1_range_t, typename sequence2_range_t>
+    //!\cond
+        requires traits_t::is_vectorised
+    //!\endcond
+    constexpr auto make_alignment_result(index_range_t && index_range,
+                                         sequence1_range_t && sequence1_range,
+                                         [[maybe_unused]] sequence2_range_t && sequence2_range)
```

Comment:
```suggestion
                                         sequence2_range_t && SEQAN3_DOXYGEN_ONLY(sequence2_range))
```
? Since it is actually never used

Reply:
It will be used at some later point. So either one of it will do as it is only temporary as long as no traceback is computed. In fact for the changes that come with the next PR this will look differently already.